### PR TITLE
Add Comment Input component

### PR DESCRIPTION
**Pull request by nguythaitinh**
Screen Shot 2022-09-03 at 05 08 09

### CODE REVIEW

File: src/components/CommentInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { useInput } from '../../utils/hook';
+
+interface Props {
+	style?: ViewStyle;
+	containerStyle?: ViewStyle;
+}
+
+export const CommentInput: FC = ({
+	style = {},
+	containerStyle = {},
```
Comment (fluctlight-kayaba) [inline src/components/CommentInput/index.tsx:13]: it's ok to let `style` and `containerStyle` empty (`undefined`), skip the empty Object fallback.

File: src/components/CommentInput/index.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}: Props) => {
```
Comment (fluctlight-kayaba) [inline src/components/CommentInput/index.tsx:14]: this`Props` should be in `CommentInput: FC<Props>` instead.

File: src/components/CommentInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+			/>
+		</View>
+	);
+};
+
+export default CommentInput;
+
+const styles = StyleSheet.create({
+	container: {
+		width: '100%',
```
Comment (fluctlight-kayaba) [inline src/components/CommentInput/index.tsx:34]: not really sure if `width: '100%'` is useful?

File: src/screens/Dashboard/ThreadList/ThreadItem/index.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React, { FC, memo } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { Text } from '@metacraft/ui';
+import CommentInput from 'components/CommentInput';
 
 import BellIcon from '../../../../components/icons/feather/Bell';
 import PinIcon from '../../../../components/icons/feather/Pin';
```
Comment (fluctlight-kayaba) [inline src/screens/Dashboard/ThreadList/ThreadItem/index.tsx:7]: use aliased import, just like `CommentInput` above:
- `import BellIcon from 'components/icons/feather/Bell'`;
- `import PinIcon from 'components/icons/feather/Pin'`;

File: src/components/CommentInput/index.tsx (hunk 1 above)
Comment (nguythaitinh) [reply src/components/CommentInput/index.tsx]: I've fixed it, pls check again

File: src/screens/Dashboard/ThreadList/ThreadItem/index.tsx (hunk 4 above)
Comment (nguythaitinh) [reply src/screens/Dashboard/ThreadList/ThreadItem/index.tsx]: I've fixed it, pls check again

File: src/components/CommentInput/index.tsx (hunk 2 above)
Comment (nguythaitinh) [reply src/components/CommentInput/index.tsx]: I've fixed it, pls check again

File: src/components/CommentInput/index.tsx (hunk 3 above)
Comment (nguythaitinh) [reply src/components/CommentInput/index.tsx]: it's not useful, i've remove it

File: src/screens/Dashboard/ThreadList/ThreadItem/index.tsx [hunk 5]
```diff
@@ -1,9 +1,10 @@
 import React, { FC, memo } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { Text } from '@metacraft/ui';
+import CommentInput from 'components/CommentInput';
+import BellIcon from 'components/icons/feather/Bell';
+import PinIcon from 'components/icons/feather/Pin';
```
Comment (fluctlight-kayaba) [inline src/screens/Dashboard/ThreadList/ThreadItem/index.tsx:6]: oops, my mistake -- we could not use `aliased` import.
It must be relative path, like `../../../../components/icons/feather/Bell`.

Comment (nguythaitinh) [reply src/screens/Dashboard/ThreadList/ThreadItem/index.tsx]: i've fixed, pls check again

**Review by fluctlight-kayaba — APPROVED** [review]

**Pull request merged**